srcpkgs/cnping/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'cnping'
+pkgname=cnping
+version=1.0.0
+revision=1
+makedepends="libXinerama-devel libX11-devel"
+depends="libcap-progs"
+short_desc="Minimal Graphical IPV4 Ping Tool"
+maintainer="Jan Christian Grünhage <jan.christian@gruenhage.xyz>"
+license="MIT, BSD-3-Clause"
+homepage="https://github.com/cnlohr/cnping"
+distfiles="https://github.com/cnlohr/cnping/archive/${version}.tar.gz"
+checksum=43e4737e27a2ab5bd8f90c73efa0ce998c2cc1d767e4948f50ded0d29e683e2d
+
+do_build() {
+	make
```

Comment:
`build_style=gnu-makefile` instead of defining do_build?  And you should always append at least `${makejobs}` to make

Reply:
fixed